### ISSUE TITLE
Being on fire while wearing "Advanced NanoChestPlate" consumes Empty Cells

### ISSUE THREAD

**Issue by Fanasuul** [GTNewHorizons/GT-New-Horizons-Modpack#15529]
### Your GTNH Discord Username

Fanasuul#6878

### Your Pack Version

2.5.1

### Your Server

Private server

### Java Version

Java 20

### Type of Server

Don't know

### Your Expectation

I was casually swimming in lava and noticed empty cells missing from my inventory.

### The Reality

Through trial and error I found the  "Advanced NanoChestPlate" to be the source of the missing cells, it happens as soon as I wear it so long as I am on fire. It happens in a fast pace like one cell per tick. They seem to simply void but maybe I am missing something.

### Your Proposal

No more cell voiding while lava swimming.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Data-Shark1** [comment]
Happens for me also, but just a visual bug. Clicking the stack of cells in my inventory while on fire, or after, causes the cells to reappear back to the original stack size.

**Comment by LazyFleshWasTaken** [comment]
maybe its trying to consume water cells to put out fire?

**Comment by rarblaster919** [comment]
I see this happen too. It's only when I'm wearing the nano armor, and if I click the cells I can usually recover them, but if I don't have the chance to do so, it WILL consume several, if not all. There's only a 3-4 second grace period to click them to save them. It's an annoying thing and by the time I figured out what was happening over weeks of play, I'd probably lost 4 stacks of cells. Just kept thinking I put them in a chest and forgot, but then I noticed it actively happening.
